core/platform/taskqueue/gae_taskqueue_services_test.py

```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the GAE taskqueue API wrapper."""
+
+import json
+import operator
+
+from core.platform.taskqueue import gae_taskqueue_services as taskqueue_services
```

Comment:
Please don't use `as`. Instead directly use `gae_taskqueue_services`.

Reply:
Just a note that this particular one is fine (and generally for stuff in core.platform only) -- we do similar things in other _test.py files. This is because we want stuff outside the platform code files to be as platform-agnostic as possible.